src/Runner.hs

```diff
@@ -124,9 +124,11 @@ runModule repl (Module module_ setup examples) = do
   where
     reload :: IO ()
     reload = do
+#if __GLASGOW_HASKELL__ < 707
       -- NOTE: It is important to do the :reload first!  There was some odd bug
-      -- with a previous version of GHC (7.4.1?).
+      -- with a previous versions of GHC (7.6.3 and older).
       void $ Interpreter.safeEval repl   ":reload"
+#endif
       void $ Interpreter.safeEval repl $ ":m *" ++ module_
```

Comment:
For all I remember, the purpose of the `:reload` is to clear the scope, so that code examples are independent of each other.  The comment merely meanst that the `:reload` should happen before the `:m *`.  This is the workaround for some weird GHC bug.  As far as I remember only one minor version of GHC was affected by this.